client/src/app/util/__tests__/generateImageSpec.js

```diff
@@ -29,6 +30,14 @@ describe('util - generateImage', function() {
       expect(image).to.exist;
     });
 
+
+    it('should automatically handle large images <' + type + '>', function() {
+
+      const image = generateImage(type, outOfBoundsSVG);
+
+      expect(image.length).to.be.greaterThan(6);
+    }).timeout(5000); // downscaling may exceed the default timeout 2000ms.
```

Comment:
is the timeout a problem in the test? maybe we can wrap that into a promise inside the test case and make it async...

Reply:
We seem to face the same problem before:
Look at [here](https://github.com/camunda/camunda-modeler/blob/8a7a941870f653a5581f898297a324ef34c142dd/client/src/app/tabs/bpmn/__tests__/BpmnEditorSpec.js#L234)

I basically applied the same solution. Creating a new canvas/writing image on it each time we downscale is a bit costly on CI machines.

Reply:
Is there a benefit to making this async?

Reply:
I see. I just thought about making it async to ensure the timeout never is caught.

Reply:
If `5000ms` is enough to ensure the test'll never fail on CI, I'd be fine with that.